dev/source/docs/raspberry-pi-via-mavlink.rst

```diff
@@ -189,7 +194,7 @@ RPi type:
 ::
 
     sudo -s
-    mavproxy.py --master=/dev/ttyAMA0 --baudrate 57600 --aircraft MyCopter
+    mavproxy.py --master=/dev/ttyAMA0 --baudrate 921600 --aircraft MyCopter
```

Comment:
agree event if not all board support those baudrate

Reply:
Agreed!